modules/database/sqlite/activate.php

```diff
@@ -41,6 +41,12 @@
 		}
 	}
 
+	// Do not automatically set up SQLite DB on a multisite as that is notably
+	// more complex. Potentially it can be added in the future.
+	if ( is_multisite() ) {
+		return;
+	}
```

Comment:
Should we also add a check for `is_multisite` in the `can-load.php` file? 🤔

Reply:
If we add the `is_multisite` check in the `can-load.php` file, we don't need to add it to the `activate.php` file. WDYT?

Reply:
@aristath @mukeshpanchal27 @ankitrox This shouldn't be necessary. It's not that the entire module doesn't work on multisite, it's just that the automatic installation doesn't work with multisite.